opennebula/shared_schemas.go

```diff
@@ -86,6 +87,38 @@ func diskFields(customFields ...map[string]*schema.Schema) map[string]*schema.Sc
 			Type:     schema.TypeString,
 			Optional: true,
 		},
+		"type": {
```

Comment:
This name is confusing. I suggest `volatile_disk_type` instead

Reply:
It's already in disk section, would `volatile_type` be ok ?

Reply:
yes